test_runner/logical_repl/test_log_repl.py

```diff
@@ -3,14 +3,17 @@
 """
 
 import hashlib
+import json
 import time
 
 import clickhouse_connect
 import psycopg2
 import pytest
+import requests
 from fixtures.log_helper import log
 from fixtures.neon_fixtures import RemotePostgres
 from fixtures.utils import wait_until
+from kafka import KafkaConsumer  # type: ignore
```

Comment:
Instead of `# type: ignore` we can add it to this list: https://github.com/neondatabase/neon/blob/e7477855b787f3f44a8e46c550eeed009ec89f0c/pyproject.toml#L71-L79

Reply:
Good catch, thanks, done